### ISSUE TITLE
Multi-hop Query Answering Uses Logits instead of likelihood scores [0,1]

### ISSUE THREAD

**Issue by sapkotaruz11** [dice-group/dice-embeddings#400]
## Problem

The multi-hop query answering pipeline currently composes **raw logits** by default, T-norm semantics (as used in CQD/LitCQD-style fuzzy composition) assume inputs are **likelihood scores in `[0,1]`**.  
Argument to modify/ update `use_logits` is missing from `answer_multi_hop_query` method  in `knowledge_graph_embeddings.py`

**Comment by sapkotaruz11** [comment]
@Demirrr Do you have some suggestions regarding this?

**Comment by Demirrr** [comment]
Please implement the option for using the logits
